code_generation/producers/taus.py

```diff
@@ -6,12 +6,24 @@
 # Set of producers used for selection of good taus
 ####################
 
+TauPtCorrection_old = Producer(
```

Comment:
I  would suggest to rename `TauPtCorrection_old` to something like `TauPtCorrection_byValue`

Reply:
Done